src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+
+public class OpeningPeriod {
```

Comment:
What is an opening period? it does not appear to be a period of time.

Reply:
The term is confusing, but apparently that is what a combination of `OpeningDay` and `date` is called in terms of `mod-calendar`:
/calendar/**periods**:
```
"openingPeriods": [
    {
      "openingDay": {
        "openingHour": [
          {
            "startTime": "07:00",
            "endTime": "13:00"
          }
        ],
        "allDay": false,
        "open": true,
        "exceptional": false
      },
      "date": "2020-01-03T00:00:00.000+0000"
    }
  ],
  "totalRecords": 1
}
```

Reply:
Ok, thank you for explaining this